setup.py

```diff
@@ -18,9 +18,12 @@
 from collections.abc import Iterator
 from typing import Any
 
+from pybind11.setup_helpers import ParallelCompile
 from setuptools import Extension, setup
 from setuptools.command.build_ext import build_ext
 
+ParallelCompile("MAX_CONCURRENCY").install()
```

Comment:
```suggestion
ParallelCompile("MAX_CONCURRENCY", default=4).install()
```
Actually, from our documentation, this should be 4.

https://pillow.readthedocs.io/en/stable/installation/building-from-source.html#build-options
> Config setting: -C parallel=n. Can also be given with environment variable: MAX_CONCURRENCY=n. Pillow can use multiprocessing to build the extension. Setting -C parallel=n sets the number of CPUs to use to n, or can disable parallel building by using a setting of 1. By default, it uses 4 CPUs, or if 4 are not available, as many as are present.